lib/jor1k.js

```diff
@@ -92,7 +97,7 @@ else
     LoadBinaryResource = function(url, OnSuccess, OnError)
     {
         //console.log(url);
-        require("fs")["readFile"](url, function(err, data)
+        import("node:" + "fs").then(fs => fs["readFile"](url, function(err, data)
```

Comment:
Do we need to concatenate "node:" and module name for `import()` every time?

Reply:
Yeah, otherwise closure compalins it cant find the module.  Making it an expression like this seems to suppress that.